src/hooks/context/hook.spec.ts

```diff
@@ -131,3 +131,31 @@ describe('useContextFile().getContext', () => {
 		expect(response instanceof SpecificationFile).toBeTruthy();
 	});
 })
+
+describe('useSpecFile should ', () => {
+	it('Load spec file from --file flag', () => {
+		const { specFile, error } = useSpecfile({ file: './test/specification.yml' });
+		expect(error).toBeUndefined();
+		expect(specFile instanceof SpecificationFile).toBeTruthy();
+	});
+	it('Load spec file from --context flag', () => {
+		testingVariables.createDummyContextFile();
+		const { specFile, error } = useSpecfile({ context: 'home' });
+		expect(error).toBeUndefined();
+		expect(specFile instanceof SpecificationFile).toBeTruthy();
+	});
+	it('Load spec file from current context', () => {
+		testingVariables.createDummyContextFile();
+		const { specFile, error } = useSpecfile({});
+		expect(error).toBeUndefined();
+		expect(specFile).toBeDefined();
+	});
+	it.skip('Load spec file from working directory', () => {
+		testingVariables.createSpecFileAtWorkingDir();
+		testingVariables.deleteDummyContextFile();
+		const { specFile } = useSpecfile({});
+		expect(specFile).toBeDefined();
+
+		testingVariables.deleteSpecFileAtWorkingDir();
+	});
```

Comment:
```suggestion
	it.skip('Load spec file from working directory', () => {
		testingVariables.createSpecFileAtWorkingDir();
		testingVariables.deleteDummyContextFile();
		const { specFile } = useSpecfile({});
		expect(specFile).toBeDefined();

		testingVariables.deleteSpecFileAtWorkingDir();
	});
```

Remove that test case or enable it :)

Reply:
This is the issue I was getting, This test case should pass but it is failing. This is happening because `useSpecFile` is throwing an error that it cannot auto-detect `asyncapi.yml` in the working directory. But is able to auto-detect when I am using it. and also the test case is passing when am testing the underlying function that has the logic to autodetect the `asyncapi.yml` file in the working directory.

Reply:
You can still play with this test :) But if you will spend more that 1 hour, please add comment why this test is skipped and make followup issue for that :)

Reply:
Yeah, I updated the test to run accordingly.